client/signup/steps/import-url/index.jsx

```diff
@@ -0,0 +1,124 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { debounce, flow, get } from 'lodash';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
+//import FormInputValidation from 'components/forms/form-input-validation';
+import { fetchIsSiteImportable, setNuxUrlInputValue } from 'state/importer-nux/actions';
```

Comment:
What if we just put these in `state/site-importer` & use a `Signup` in the names where appropriate?

If not, can we call this `state/importer-signup` instead of `state/importer-nux`?

Reply:
Hey @jblz, sorry this one slipped me by for some reason.

I'm wary of building different parts of the UI with conflated state. There are a couple of areas in Calypso where forms in different parts of the UI share state, so editing a field in one place updates it in another - inline-helps contact form and the contact form found in `/help/` is one example. It's not of much consequence but I'm fairly sure that that instance came about from oversight rather than intention.

That said, it might actually be wanted behaviour here. If the url field has `x` content in the signup flow and somehow the customer ends up in the regular importer screen, they would likely only benefit from having that input mirrored there.
This certainly does apply for an import thats already been kicked off, so that the whole app can reflect the state of that import session.

I'll make a new ticket to address this, because I do think you're right in putting all of this under `state/site-importer`... If we don't want the form fields to share state, we can just create new state slices within `state/site-importer`

Reply:
Just a note to make sure we handle the renaming in a future PR.